src/pages/index.tsx

```diff
@@ -19,6 +19,37 @@ export default function Home() {
     !loading && getLogedInUser();
   }, [user, loading]);
 
+  useEffect(() => {
+    // Fetch the data from the GitHub URL
+    fetch(
```

Comment:
We should post this into the database as part of the migration, we then have a skills api that we can call to get existing skils based on what a user types

Reply:
the skills api is available